dev/docker-compose.yml

```diff
@@ -1,33 +1,29 @@
 version: '3'
 services:
     mattermost:
+        container_name: mattermost
         # This can be updated with docker tags from https://hub.docker.com/r/mattermost/mattermost-enterprise-edition/tags
         image: mattermost/mattermost-enterprise-edition:6.6
-
         ports:
-            - "8066:8066"
+            - 8065:8065
```

Comment:
This was originally `8066` so you could also run the typical Go server on port `8065`. 0/5 on if it should be kept that way

Reply:
Was that the reason why? I found it odd that this was sitting on `8066` since we use `8065` everywhere.